### PR TITLE
Updates demo library script to take in multiple slugs and write output to script

### PR DESCRIPTION
**Pull request by vickygos**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "open-rarity"
-version = "0.2.1-beta"
+version = "0.3.1-beta"
```
Comment (vickygos) [inline pyproject.toml:3]: This was for https://github.com/OpenRarity/open-rarity/pull/39

File: open_rarity/models/collection.py [hunk 2]
```diff
@@ -67,14 +66,51 @@ def __init__(
         tokens: list[Token],
         name: str | None = "",
     ):
+        self._tokens = tokens
```
Comment (vickygos) [inline open_rarity/models/collection.py:69]: this file's changes was actually for: https://github.com/OpenRarity/open-rarity/pull/39
but i accidentally pushed to wrong PR and merged the other one. cc @theelderbeever

Comment (vickygos) [reply open_rarity/models/collection.py:69]: This is to disallow users to reset @tokens

Comment (theelderbeever) [reply open_rarity/models/collection.py:69]: Great, acknowledged

**Review by theelderbeever — APPROVED** [review]
Lgtm

File: pyproject.toml (hunk 1 above)
Comment (impreso) [reply pyproject.toml:3]: Should be 0.3.0-beta

Comment (vickygos) [reply pyproject.toml]: fixed

**Pull request merged**